src/components/SharedComponents/PhotoCount.js

```diff
@@ -3,23 +3,29 @@
 import { View } from "components/styledComponents";
 import * as React from "react";
 import Svg, { Path, Text } from "react-native-svg";
+import { viewStyles } from "styles/sharedComponents/photoCount";
+import colors from "styles/tailwindColors";
 
 type Props = {
   count: number,
   size?: number,
-  shadow: boolean,
+  shadow?: boolean,
 };
 
 const PhotoCount = ( { count, size, shadow }: Props ): React.Node => {
-  if ( count <= 0 || count > 99 ) {
+  if ( count === 0 ) {
     return null;
   }
 
+  let photoCount = count;
+  if ( count > 99 ) {
+    photoCount = 99;
+  }
+
   return (
     <View
-      className={`drop-shadow-md w-[${String( size )}px] h-[${String( size )}px] ${String(
-        shadow && "shadow"
-      )}`}
+      style={[{ height: size, width: size }, shadow && viewStyles.shadow]}
```

Comment:
We want to rely mostly on Tailwind/Nativewind for styling of our app. So, since you checked that drop-shadow is not possible to add with class names, you can leave that here in style, but change everything back to class name. Also, maybe change the size probe to `classNameSize`, then you don't need to create a string from the size number here, just concatenate it.

Reply:
Hi, we actually have added a default style for drop-shadow in a different pull request. Please import the `dropShadow` style from here `styles/global.js`.